vars/terraformDeployJob.groovy

```diff
@@ -96,12 +96,14 @@ def call(Map config) {
             steps {
               //Terraform doesn't deploy using tags until https://national-archives.atlassian.net/browse/TDR-1229 is implemented
               //Ensure that the tagging between Jenkins intg and prod instances remain in sync by not tagging if deploying to staging and prod
-              if (config.stage == "intg" || config.stage == "mgmt") {
-                sh "git tag ${versionTag}"
-                sshagent(['github-jenkins']) {
-                  sh("git push origin ${versionTag}")
+              script {
+                if (config.stage == "intg" || config.stage == "mgmt") {
+                  sh "git tag ${versionTag}"
+                  sshagent(['github-jenkins']) {
+                    sh("git push origin ${versionTag}")
+                  }
                 }
-              }              
+              }                           
```

Comment:
Looks like lots of trailing whitespace has been added?

Reply:
Removed